org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/context/BundleContextExtension.java

```diff
@@ -70,8 +70,8 @@
  */
 public class BundleContextExtension implements BeforeAllCallback, BeforeEachCallback, ParameterResolver {
 
-	public static final String		BUNDLE_CONTEXT_KEY	= "bundle.context";
-	public static final String		INSTALL_BUNDLE_KEY	= "install.bundle";
+	public static final String	BUNDLE_CONTEXT_KEY	= "bundle.context";
+	public static final String	INSTALL_BUNDLE_KEY	= "install.bundle";
```

Comment:
Should be `BUNDLE_INSTALLER_KEY = "bundle.installer"`

Reply:
thank you